cmd/notary/tuf.go

```diff
@@ -249,6 +253,21 @@ func getTargetHashes(t *tufCommander) (data.Hashes, error) {
 	return targetHash, nil
 }
 
+// Open and read a file containing the targetCustom data
+func getTargetCustom(targetCustomFilename string) (json.RawMessage, error) {
+	var nilCustom json.RawMessage
+	targetCustomFile, err := os.Open(targetCustomFilename)
```

Comment:
Non-blocking:  `ioutil.ReadFile` might save some typing here, because it will open and close the file for you.